interface/billing/sl_eob_search.php

```diff
@@ -528,7 +526,14 @@ function upload_file_to_client_pdf($file_to_send, $aPatFirstName = '', $aPatID =
                 continue;
             }
         } else {
-            fwrite($fhprint, make_statement($stmt));
+            if ($inv_pid[$inv_count] != $inv_pid[$inv_count+1]) {
+                $tmp = make_statement($stmt);
+                if(empty($tmp)) {
+                    $tmp = xlt("This EOB Id: $inv_pid[$inv_count] Encounter: $stmt[encounter] does not meet minumum print requirements setup in Globals or there is an unknown error."."\n");
```

Comment:
This will be better for translation(otherwise, the variables will mean this will never be translated):
``````
$tmp = xlt("This EOB item does not meet minimum print requirements setup in Globals or there is an unknown error.") . " " . xlt("EOB Id") . ":" . $inv_pid[$inv_count] . " " . xlt("Encounter") . ":" . $stmt[encounter] . "\n";